deps.bzl

```diff
@@ -7071,11 +7071,11 @@ def install_buildbuddy_dependencies(workspace_name = "buildbuddy"):
     )
 
     http_file(
-        name = "com_github_buildbuddy_io_soci_snapshotter-soci-store-v0.0.7-linux-amd64",
-        urls = ["https://storage.googleapis.com/buildbuddy-tools/binaries/soci-snapshotter/soci-store-v0.0.7-linux-amd64"],
-        sha256 = "1149c110188a552982498e29fe52ae6c65a91d389e95a98d00a7b8dd771e109e",
+        name = "com_github_buildbuddy_io_soci_snapshotter-soci-store-v0.0.8-linux-amd64",
+        urls = ["https://storage.googleapis.com/buildbuddy-tools/binaries/soci-snapshotter/soci-store-v0.0.8-linux-amd64"],
+        sha256 = "978febe41046d9338e766ddd0a8f3f46d798b7f5b7a537840f4d9c2f814a759a",
         executable = True,
-        downloaded_file_path = "soci-store-v0.0.7-linux-amd64",
+        downloaded_file_path = "soci-store-v0.0.8-linux-amd64",
```

Comment:
(unrelated to this PR) does the soci-snapshotter go repo have a `go_binary` target generated by gazelle that we can depend on instead? I think that'd let us avoid having to upload stuff to GCS

Reply:
Yeah, it does. I have a branch in progress that links it into the executor and runs it in-process, but will pursue your recommendation if that doesn't work for some reason.